hudi-sync/hudi-sync-common/src/main/java/org/apache/hudi/sync/common/HoodieMetaSyncOperations.java

```diff
@@ -186,16 +188,20 @@ default void updateLastCommitTimeSynced(String tableName) {
 
   /**
    * Update the table properties in metastore.
+   *
+   * @return true if properties updated.
    */
-  default void updateTableProperties(String tableName, Map<String, String> tableProperties) {
-
+  default boolean updateTableProperties(String tableName, Map<String, String> tableProperties) {
+    return false;
   }
 
   /**
    * Update the SerDe properties in metastore.
+   *
+   * @return true if properties updated.
    */
-  default void updateSerdeProperties(String tableName, Map<String, String> serdeProperties) {
-
+  default boolean updateSerdeProperties(String tableName, Map<String, String> serdeProperties, boolean useRealtimeFormat) {
```

Comment:
Unrelated to this patch - i'm wondering if we should change the signature of all methods that alter table? Make all of them return a boolean and take some action. For example, `updateLastCommitTimeSynced` is called after we have update table props, schema, etc. Supposr `updateLastCommitTimeSynced` failed, then an exception is thrown but are previous updates (of table props, schema) in this sync rolled back?

Reply:
yea all metasync api should be standardized with a return flag or a pojo containing sync result. it's just not that critical to change right now.

Reply:
Got it. Can you please create a JIRA to track? We can work on it sometime later after 0.14 critical items.

Reply:
done https://issues.apache.org/jira/browse/HUDI-6250